spring-boot-modules/spring-boot-data-3/src/test/java/com/baeldung/nopropertyfound/PersonRepositoryTest.java

```diff
@@ -0,0 +1,28 @@
+package com.baeldung.nopropertyfound;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import com.baeldung.nopropertyfound.model.Person;
+import com.baeldung.nopropertyfound.repository.PersonRepository;
+
+@DataJpaTest
+class PersonRepositoryTest {
+
+    @Autowired
+    private PersonRepository personRepository;
+
+    @Test
+    void givenQueryMethod_whenUsingValidProperty_theCorrect() {
+
+        Person person = personRepository.findByFirstName("Azhrioun");
+
+        assertNotNull(person);
```

Comment:
Call to 'assertNotNull()' from 'Assert' should be replaced with a call to a method from 'org.junit.jupiter.api.Assertions'